bedrock/mozorg/templates/mozorg/moss/secure-open-source.html

```diff
@@ -0,0 +1,232 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/moss/base.html" %}
+
+{% block page_title %}
+  {{ _('Secure Open Source') }}
+  &mdash;
+  {{ super() }}
+{% endblock %}
+
+{% block body_id %}moss-secure-open-source{% endblock %}
+
+{% block page_heading %}
+  {{ _('Secure Open Source') }}
+{% endblock %}
+
+{% block page_content %}
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('About') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    The Secure Open Source ("SOS") track of MOSS supports security audits for
+    open source software projects, and remedial work to rectify the problems
+    found.
+  {% endtrans %}
+
+  {% trans url1='https://wiki.mozilla.org/MOSS/Secure_Open_Source/Completed' %}
+    You can read about the
+    <a href="{{ url1 }}">audits we’ve completed so far</a>.
+  {% endtrans %}
+  </p>
+
+  <div class="moss-ctas">
+    <p class="moss-copy">{{ _('Ready to suggest a recipient?') }}</p>
+
+    <a href="https://docs.google.com/forms/d/1f0xSg9XM8v7YGdZ_FzeE67ggckbAsg6sH1mpQ4buTQE/viewform" class="button" rel="external">{{ _('Nominate Now') }}</a>
+  </div>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">
+    {{ _('What your project should include') }}
+  </h2>
+
+  <p class="moss-copy">
+    {{ _('The SOS Fund has a very limited set of solid rules:') }}
+  </p>
+
+  <ul class="moss-bullet-list">
+    <li>
+      <p class="moss-copy">
```

Comment:
Maybe the nested paragraphs aren't necessary, could this just be `<li class="moss-copy">` ?